src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/commands/mobile/set_interior_vehicle_data_request.h

```diff
@@ -39,6 +39,8 @@ namespace rc_rpc_plugin {
 namespace app_mngr = application_manager;
 
 namespace commands {
+typedef std::pair<std::string, mobile_apis::Result::eType> module_capability;
```

Comment:
@KVGrygoriev according to our code style 
https://google.github.io/styleguide/cppguide.html#Type_Names
module_capability --> ModuleCapability

Reply:
@EKuliiev, see 8629eb6a49d3f0c6008f851492cbba3cd43ff9e8